examples/ollama-openwebui/compose.yaml

```
@@ -0,0 +1,46 @@
+  olla:
+    image: ghcr.io/thushan/olla:latest
+    container_name: olla
+    restart: unless-stopped
+    ports:
+      - "40114:40114"
+    volumes:
+      # Overide the default with our custom configuration
+      - ./olla.yaml:/app/config.yaml:ro
+      - ./.container/olla/logs:/app/logs:ro
+    healthcheck:
+        test: ["CMD", "wget", "--quiet", "--tries=1", "--spider", "http://localhost:40114/internal/health"]
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)